src/Hydra/Serializer/CollectionNormalizer.php

```diff
@@ -89,7 +89,7 @@ public function normalize($object, $format = null, array $context = [])
         $data['hydra:member'] = [];
 
         // IriOnly's adaptations
-        if (true === $context[self::IRI_ONLY] ?? $this->defaultContext[self::IRI_ONLY]) {
+        if (isset($context[self::IRI_ONLY]) && true === $context[self::IRI_ONLY] ?? $this->defaultContext[self::IRI_ONLY]) {
```

Comment:
`isset` is useless here as the [null coalesce operator `??` ](https://www.php.net/manual/en/language.operators.comparison.php#language.operators.comparison.coalesce) does it already. Maybe we can use:
```
true === $context[self::IRI_ONLY] ?? $this->defaultContext[self::IRI_ONLY] ?? false
```

Reply:
I can't pass the unit tests if I do that. I got an "Undefined index: iri_only".